nodebuilder/share/share.go

```diff
@@ -26,9 +29,8 @@ var _ Module = (*API)(nil)
 //
 //go:generate mockgen -destination=mocks/api.go -package=mocks . Module
 type Module interface {
-	// SharesAvailable subjectively validates if Shares committed to the given Root are available on
-	// the Network.
-	SharesAvailable(context.Context, *share.Root) error
+	// SharesAvailable subjectively validates if Shares committed to the given Root are available on the Network.
+	SharesAvailable(context.Context, *share.Root, ...peer.ID) error
```

Comment:
Hmm, it's not necessary to bring peering to the module level, so let's avoid that and simplify things